spec/api-autoupdater-darwin-spec.ts

```diff
@@ -68,14 +68,14 @@ ifdescribe(shouldRunCodesignTests)('autoUpdater behavior', function () {
       await withTempDirectory(async (dir) => {
         const secondAppPath = await copyMacOSFixtureApp(dir, fixture);
         const appPJPath = path.resolve(secondAppPath, 'Contents', 'Resources', 'app', 'package.json');
-        await fs.writeFile(
+        fs.writeFileSync(
           appPJPath,
-          (await fs.readFile(appPJPath, 'utf8')).replace('1.0.0', version)
+          fs.readFileSync(appPJPath, 'utf8').replace('1.0.0', version)
         );
         const infoPath = path.resolve(secondAppPath, 'Contents', 'Info.plist');
-        await fs.writeFile(
+        fs.writeFileSync(
           infoPath,
-          (await fs.readFile(infoPath, 'utf8')).replace(/(<key>CFBundleShortVersionString<\/key>\s+<string>)[^<]+/g, `$1${version}`)
+          fs.readFileSync(infoPath, 'utf8').replace(/(<key>CFBundleShortVersionString<\/key>\s+<string>)[^<]+/g, `$1${version}`)
```

Comment:
Why the change from async here?

Reply:
Because the extra cognitive complexity of  `await fs.asyncFuncOne(await fs.asyncFuncTwo())` doesn't seem to buy us anything